pkg/ccl/backupccl/backup_processor_planning.go

```diff
@@ -69,17 +64,35 @@ func distBackup(
 		execCtx.User(),
 		execCtx.ExecCfg(),
 	)
+}
+
+// distBackup is used to plan the processors for a distributed backup. It
+// streams back progress updates over progCh, which is used to incrementally
+// build up the BulkOpSummary.
+func distBackup(
+	ctx context.Context,
+	execCtx sql.JobExecContext,
+	progCh chan *execinfrapb.RemoteProducerMetadata_BulkProcessorProgress,
+	backupSpecs map[roachpb.NodeID]*execinfrapb.BackupDataSpec,
+) error {
+	ctx = logtags.AddTag(ctx, "backup-distsql", nil)
+	var noTxn *kv.Txn
+
+	dsp := execCtx.DistSQLPlanner()
+	evalCtx := execCtx.ExtendedEvalContext()
+
+	// We don't return the compatible nodes here since PartitionSpans will
+	// filter out incompatible nodes.
+	planCtx, _, err := dsp.SetupAllNodesPlanning(ctx, evalCtx, execCtx.ExecCfg())
```

Comment:
hm, I _think_ we should be re-using the same planing ctx instead of making a new one? I wonder if we just want to move the call to SetupAllNodesPlanning the caller and pass it to each of these?

Reply:
Yam, I needed to do some general cleanup of the planCtx and the dsp. I _think_ things are in order now but want to double check with fresh eyes.